### PR TITLE
fix: add Shehroz Irfan

### LINKED ISSUE #1594: [PROFILE ADDITION] <Shehroz Irfan>
### Profile screenshot

shehroz-irfan-awesome-github-profile

### PR DESCRIPTION
**Pull request by ShehrozIrfan**
resolves #1594 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/ShehrozIrfan.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/36946330/209469655-979fe365-cf4a-43ea-a4c2-c925018b9e00.png",
```
Comment (Panquesito7) [inline profiles/ShehrozIrfan.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/209481287-b1a4db93-4b43-4b21-af39-60055c71488a.png",
```

Comment (ShehrozIrfan) [reply profiles/ShehrozIrfan.json]: Hey @Panquesito7 
Thanks for your feedback, I've added the updates for image.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**